tools/spectral/ipa/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "@mongodb-js/ipa-validation-ruleset",
+  "version": "1.0.0",
+  "description": "Custom validation rules for MongoDB API Standards (IPA).",
+  "keywords": [
+    "mongodb",
+    "openapi",
+    "spectral"
+  ],
+  "homepage": "https://github.com/mongodb/openapi/tree/main/tools/spectral/ipa#readme",
```

Comment:
[Nit/Separate for the PR] I think there is benefit of mentioning validation as part of the ipa website. Then we can use official IPA website link rather than nested repository link that might change